### ISSUE TITLE
Refactor & clarify API

### ISSUE THREAD

**Issue by adamltyson** [brainglobe/cellfinder#300]
- [ ] Keyword args for detect & classify to simplify their use independently

**Comment by EmanAbdelhaleem** [comment]
Hi @adamltyson, is this issue still an important one? 
+ Can you please help me by pointing out where to start to grasp the problem and start solving it?

**Comment by adamltyson** [comment]
It is still an important issue. However, if you're not very familar with cellfinder, it may not be the best place to start contributing.

**Comment by EmanAbdelhaleem** [comment]
Yep, I am still very new to cellfinder, I have read the developer guide and I am trying to explore the codebase nowadays, could you help me with choosing an appropriate issue, please? Preferably related to the "improve cellfinder's classification" project.

Thank you!

**Comment by adamltyson** [comment]
Hi @EmanAbdelhaleem, would you mind asking all GSoC questions on the [Neuroinformatics Unit Zulip](https://neuroinformatics.zulipchat.com/#narrow/channel/487898-GSoC) so that everyone watching this repo isn't notified by our chats? Thanks!

**Comment by EmanAbdelhaleem** [comment]
Sure, sorry for the inconvenience
